src/components/sbb-card/sbb-card.scss

```diff
@@ -0,0 +1,118 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-card-padding-inline: var(--sbb-spacing-responsive-xxs);
+  --sbb-card-padding-block-start: var(--sbb-spacing-fixed-3x);
+  --sbb-card-padding-block-end: var(--sbb-card-padding-block-start);
+}
+
+:host([size='s']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-xxxs);
+}
+
+:host([size='m']) {
+  --sbb-card-padding-inline: var(--sbb-spacing-responsive-s);
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-xxxs);
+}
+
+:host([size='l']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-fixed-4x);
+}
+
+:host([size='xl']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-xxs);
+}
+
+:host([size='xxl']) {
+  --sbb-card-padding-inline: var(--sbb-spacing-responsive-s);
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-s);
+}
+
+:host(.sbb-card--has-badge[size='m']),
+:host(.sbb-card--has-badge[size='l']),
+:host(.sbb-card--has-badge[size='xl']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-fixed-6x);
+}
+
+:host(.sbb-card--has-badge[size='m']) {
+  --sbb-card-padding-block-end: var(--sbb-spacing-responsive-xxxs);
+}
+
+:host(.sbb-card--has-badge[size='l']) {
+  --sbb-card-padding-block-end: var(--sbb-spacing-fixed-4x);
+}
+
+:host(.sbb-card--has-badge[size='xl']) {
+  --sbb-card-padding-block-end: var(--sbb-spacing-responsive-xxs);
+}
+
+.sbb-card {
+  position: relative;
+  overflow: hidden;
+  border-radius: var(--sbb-border-radius-4x);
+  transition: box-shadow var(--sbb-animation-duration-6x) var(--sbb-animation-easing);
+
+  &::before {
+    position: absolute;
+    background-color: transparent;
+    content: '';
+    inset: 0;
+    border-radius: var(--sbb-border-radius-4x);
+  }
+
+  &:focus-visible {
+    @include focusOutline;
+  }
+}
+
+:host([active]) {
+  .sbb-card::before {
+    border: var(--sbb-border-width-2x) solid var(--sbb-color-smoke-default);
+  }
+}
+
+.sbb-card__link {
+  display: block;
+  color: inherit;
+  text-decoration: none;
+}
+
+.sbb-card__button {
+  @include buttonReset;
+  @include text-inherit;
+
+  border-radius: var(--sbb-border-radius-4x);
+  cursor: pointer;
+}
+
+.sbb-card__button,
+.sbb-card__link {
+  width: 100%;
+  background-color: var(--sbb-color-white-default);
+  padding-inline: var(--sbb-card-padding-inline);
+  padding-block-start: var(--sbb-card-padding-block-start);
+  padding-block-end: var(--sbb-card-padding-block-end);
+}
+
+.sbb-card__content {
+  display: block;
+  transition: transform var(--sbb-animation-duration-6x) var(--sbb-animation-easing);
+}
+
+@include hover-mq($hover: true) {
+  :host {
+    .sbb-card:not([active]):hover {
```

Comment:
the active flag is no the host, right? So this seems not to work for me (but did not test myself ;) )

Reply:
Strange, i tested it (because the lint failed putting the selector on the host) and it seems to works, but now it do not, maybe the server joked on me.. fixed 💪